benchmarks/triton_kernels_benchmark/benchmark_testing.py

```diff
@@ -378,6 +380,30 @@ def run(self, show_plots=False, print_data=False, return_df=False, save_precisio
 
         return None
 
+    def run_constrained(
+        self,
+        shapes: List[Shape],
+        show_plots=False,
+        print_data=False,
+        save_precision=6,
+        mark_args=None,
+        **kwargs,
+    ) -> pd.DataFrame:
```

Comment:
We can pass them via `kwargs` since we don't work with them in this function.
```suggestion
    def run_constrained(
        self,
        shapes: List[Shape],
        **kwargs,
    ) -> pd.DataFrame:
```